webpack.config.js

```diff
@@ -105,14 +126,15 @@ const wordpressExternals = ( context, request, callback ) =>
  *
  * @see {@link https://webpack.js.org/configuration/configuration-types/#exporting-a-function}
  *
- * @param {object}  env                              additional config options
- * @param {boolean} env.externalizeWordPressPackages whether to bundle or extern the `@wordpress/` packages
- * @param {object}  argv                             given by webpack?
+ * `env` params are typically set by files in `bin/sdk/*`
```

Comment:
This comment might be confusing for anyone who isn't familiar with the SDK. Would you mind clarifying what can be found in `bin/sdk/*`? Or is there a chance we're leaking SDK functionality out of the SDK directory too much?